+opencossan/+common/+utilities/isunique.m

```diff
@@ -1,7 +1,7 @@
 function [Lunique] = isunique(Cinput)
 %ISUNIQUE  Lout = isunique(Cinput) returns true if no items are repeated in
-%the cell array Cinput   
-[~,ia,ic]=unique(Cinput);
-Lunique=length(ia)==length(ic);
+%the cell array Cinput
+[~,ia,ic] = unique(Cinput);
```

Comment:
This is the best change in the code I have ever seen. Also most necessary change in opencossan.